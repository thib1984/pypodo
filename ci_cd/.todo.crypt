gAAAAABfpvgUHmP3F-7fnL8vQgjTFTYN34rdvxJJm4f6cqRvcyfTEO7R2ebaNsBSqdze84QhcDApSF_4jibBM8XWgGe6g1HRTfvOoC9-uqxMHeXqX59CXotclTmm2PcNsOVEkzPnp8ytnOwBVl-32gYdwEWRX2bGj4GPUdcN1gKfP5CaaEM2cbUkdYA93mxFtiCi2gN3Fcf9