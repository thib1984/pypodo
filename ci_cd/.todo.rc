#grey,red,green,yellow,blue,magenta,cyan,white only ;)
[COLOR]
alert = magenta
warning = white
info = blue
index = red
task = yellow
tag = grey

[SYSTEM]
#info,warning,error
messagelevel = warning
key = admin
#not modify if you use docker!
#todofile = /tmp/.todo
#todobackupfolder = /tmp/.todo_backup/

[FONCTIONAL]
#int values
periodalert = 0
periodwarning = 7
#tags with '#' and with a coma separation
alerttags = #urgent,#urgente
#or True
autosort = True
