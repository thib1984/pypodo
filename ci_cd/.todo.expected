1 ta#urgentsk1#
2 task3 #urgent #30000101 #tag2 #veryurgent #newtag
3 task4 #tag #newtag #newtag
